prov/gni/src/gnix_ep.c

```diff
@@ -1037,6 +1037,7 @@ static int gnix_ep_control(fid_t fid, int command, void *arg)
 					ret);
 				goto err;
 			}
+			ep->enabled = 1;
```

Comment:
i noticed this field and thought we might actually need to be checking this when an EP is used for send/recv/rma/amo reqs.